common/cmake/devices.cmake

```diff
@@ -1447,16 +1456,40 @@ function(ADD_FPGA_TARGET)
     message(FATAL_ERROR "SDC and XDC constraint files cannot be provided simultaneously!")
   endif()
 
+  set(SDC_FILE "")
   if(NOT "${ADD_FPGA_TARGET_INPUT_XDC_FILE}" STREQUAL "")
     append_file_dependency(VPR_DEPS ${OUT_SDC_REL})
     get_file_location(SDC_LOCATION ${OUT_SDC_REL})
     set(SDC_ARG --sdc_file ${SDC_LOCATION})
+    set(SDC_FILE ${SDC_LOCATION})
   endif()
 
   if(NOT "${ADD_FPGA_TARGET_INPUT_SDC_FILE}" STREQUAL "")
     append_file_dependency(VPR_DEPS ${ADD_FPGA_TARGET_INPUT_SDC_FILE})
     get_file_location(SDC_LOCATION ${ADD_FPGA_TARGET_INPUT_SDC_FILE})
     set(SDC_ARG --sdc_file ${SDC_LOCATION})
+    set(SDC_FILE ${SDC_LOCATION})
+  endif()
+
+  if (${ADD_FPGA_TARGET_INSTALL_CIRCUIT})
+    add_custom_target(
```

Comment:
Not sure what this custom target is for?

Reply:
It was to build the dependency to install the eblif, but I think it is not needed actually. Removed

Reply:
Actually this target is needed when calling the install target, otherwise the eblifs do not get built.